evalml/tests/automl_tests/test_automl.py

```diff
@@ -216,6 +223,61 @@ def test_search_results(X_y_regression, X_y_binary, X_y_multi, automl_type, obje
     )
 
 
+def test_search_batch_times(caplog):
+    caplog.clear()
+    X, y = load_data(
+        path="evalml/tests/data/fraud_transactions.csv.gz",
+        index="id",
+        target="fraud",
+        compression="gzip",
+        n_rows=250,
+        verbose=True,
+    )
+    X.ww.set_types(logical_types={"provider": "Categorical", "region": "Categorical"})
+
+    X.ww["expiration_date"] = X["expiration_date"].apply(
+        lambda x: "20{}-01-{}".format(x.split("/")[1], x.split("/")[0])
+    )
+    X = infer_feature_types(
+        X,
+        feature_types={
+            "store_id": "categorical",
+            "expiration_date": "datetime",
+            "lat": "categorical",
+            "lng": "categorical",
+            "provider": "categorical",
+        },
+    )
+    X_train, _, y_train, _ = evalml.preprocessing.split_data(
```

Comment:
We can skip out on all this logic about splitting and datachecks as well. Look at something like `test_pipeline_score_raises` for an example.